test/parallel/test-console.js

```diff
@@ -1,14 +1,18 @@
 'use strict';
-require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
 assert.ok(process.stdout.writable);
 assert.ok(process.stderr.writable);
 // Support legacy API
 assert.equal('number', typeof process.stdout.fd);
 assert.equal('number', typeof process.stderr.fd);
 
-assert.throws(function() {
+assert.doesNotThrow(function() {
+  const message = /No such label 'no such label' for console\.timeEnd\(\)/;
+  const onWarning = (warning) => assert(message.test(warning.message));
+  process.once('warning', common.mustCall(onWarning));
+
```

Comment:
this could likely be rewritten and simplified to just:

```js
process.once('warning', common.mustCall((warning) => {
  assert(/no such label/.test(warning.message);
});
```

There's likely very little need to test the entire warning message. You just want to verify that the warning is being issued.